src/BAP/SimpleBTSBundle/Controller/IssueController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace BAP\SimpleBTSBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+use Oro\Bundle\SecurityBundle\Annotation\Acl;
+
+/**
+ * @Route("/issue")
+ */
+class IssueController extends Controller
+{
+    /**
+     * @Route("/", name="bap_bts.issue_index")
+     * @Template
+     * @Acl(
```

Comment:
This annotation does not have sense now because your issue entity does not acl protected.